### PR TITLE
CSB-891 Updated documentation for installation

### PR DESCRIPTION
**Pull request by lmccormack**
Updated docs

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,98 +1,107 @@
...
+To add additional content, each EPUB must be unpacked, a manifest generated, and the files manually 
 made part of the application's static files:
-* Clone [r2-shared-js](https://github.com/readium/r2-shared-js)
-* In r2-shared-js directory:
+* Clone and install [r2-shared-js](https://github.com/readium/r2-shared-js) from Github
+* Unpack EPUBs in the r2-shared-js directory
   * `npm run cli file.epub output-dir`
-* Copy the directory in output-dir to `clusive/shared/static/shared/pubs/short-name-for-new-pub`
-* Log in as a superuser, and in the "Books" page of the admin site, click "Rescan books".
+* Copy unpacked EPUB directory in output-dir to `/shared/static/shared/pubs/short-name-for-new-pub`
```
Comment (bgoldowsky) [inline INSTALL.md]: Insert: grunt build-noclean

File: src/clusive_project/settings_prod.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
 ALLOWED_HOSTS = ['clusive.cast.org',
                  'clusive.qa.cast.org',
                  'cisl-demo.qa.cast.org',
+                 'host.docker.internal',
```
Comment (bgoldowsky) [inline src/clusive_project/settings_prod.py]: Not sure why this would be needed?

File: INSTALL.md [hunk 3]
```diff
@@ -1,98 +1,107 @@
...
+  * Version 3.7.4 or later. On Mac, Homebrew is the easiest way to install
+* [virtualenv](https://virtualenv.pypa.io/en/latest/) 
+  * Highly recommended for maintaining an isolated environment and dependencies
 * [Postgres](https://www.postgresql.org/) 
-  * Used in the deployment configuration; 
-    may also be necessary for the psycopg2 dependency (Homebrew on OS X can take care of this)
-* [npm](https://www.npmjs.com/)
+  * Version 11.5 or later. Used in the deployment configuration 
+* [npm/grunt](https://www.npmjs.com/get-npm) 
   * Used as a build system to manage all other dependencies
```
Comment (mbrambilla) [inline INSTALL.md]:
```suggestion
* [Node.js](https://nodejs.org/)
  * Used for the build system and to manage all other dependencies using:
    * [npm](https://www.npmjs.com/get-npm) - Package manager
    * [Grunt](https://gruntjs.com/) - Task runner
```

Comment (bgoldowsky) [reply INSTALL.md]: If I understand correctly, npm is the prerequisite that you need to install manually.  You should get grunt installed for you when you type 'npm install'.

Comment (mbrambilla) [reply INSTALL.md]: - npm is installed when Node.js is installed.
- grunt is installed by `npm install`

File: src/clusive_project/settings_prod.py (hunk 2 above)
Comment (lmccormack) [reply src/clusive_project/settings_prod.py:20]: That was what I used when I tried to run locally using docker.  It didn't seem to be needed in the file, but I added it as a documentation.

**Pull request merged**